Aliases/prestosql

```diff
@@ -0,0 +1 @@
+../Formula/trino.rb
```

Comment:
The alias isn't needed once we have it in `formula_renames.json`.

Reply:
fixed